external/js_of_ocaml/runtime/wasm/fail.wat

```diff
@@ -42,6 +42,13 @@
          (array.get $block (global.get $caml_global_data)
                     (global.get $OUT_OF_MEMORY_EXN))))
 
+   (global $OUT_OF_FIBERS_EXN i32 (i32.const 0))
```

Comment:
It looks like this would need to be a unique id, but actually we can skip adding the exception in JS/wasm since they don't support fibers (perhaps we'll use wasmfx eventually)